modules/virt-about-preallocation.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc
+
+[id="virt-about-preallocation_{context}"]
+= About preallocation
+
+The Containerized Data Importer (CDI) can use the QEMU preallocate mode for data volumes in order to improve write performance. Preallocation mode can be used for importing and uploading operations and when creating blank data volumes.
+
+If preallocation is enabled, the CDI uses the better preallocation method depending on the underlying file system and device type.
+
+`fallocate`::
+If the file system supports it, CDI will use the `fallocate` call of the operating system to preallocate space using the `posix_fallocate` function, which allocates blocks and marks them as uninitialized. 
+
+`full`::
+If `fallocate` mode can not be used, `full` mode allocates space for the image by writing data to the underlying storage. Depending on the storage location, all the empty allocated space might be set as a zero.
```

Comment:
s/can not/cannot

"set as a zero" seems like iffy phrasing, but I might not be familiar with this kind of storage language.